frontend/packages/kubevirt-plugin/src/components/modals/nic-modal/nic-modal.tsx

```diff
@@ -102,9 +102,13 @@ export const Network: React.FC<NetworkProps> = ({
       title="Network"
       fieldId={id}
       isRequired
-      isLoading={nadsLoading}
-      validationMessage={nadsLoadError}
-      validationType={nadsLoadError && ValidationErrorType.Error}
+      isLoading={nadsLoading && !nadsLoadError}
+      validationMessage={
+        nadsLoadError || networkChoices.length === 0
+          ? 'No Network Attachment Definitions available. Contact your system admin for additional support'
```

Comment:
this should check the nadsLoadError properly to display such a message.

Reply:
that is the bug we are fixing, we do not want to show the raw error message, we just want to inform users that they don't have nads.
An error fetching nads may be ok here (e.g. if they don't have credentials), or not (e.g. api down), and we just want to inform them that they can't use nads, and if they think they should, they can investigate ask for admin help,

Reply:
I think it might be useful to give a small clue what is wrong.

- If it is a permission error - they know that they have to contact an admin
- if nads are empty. They can just create it themselves and they dont have to bother admin.

cc @matthewcarleton thoughts?

Reply:
+1 actionable information that is accurate is the goal so we should def provide them a path to fix the issue (especially if they can do it themselves)
If they can create one we should say 
`create a Network attachment definition` is an action we should provide from the alert rather than telling them to talk to the admin if they can.

Reply:
@matthewcarleton their are two different texts now, one for error and one for no NADs:
https://github.com/openshift/console/pull/6554/files#diff-1c2a6b849bb9ba8b02f00ccb4ee98149R99

Reply:
@matthewcarleton 
p.s. even if the problem is not missing credentials and user can create a NAD they probably need admin help because they need to know the name of the network bridge to use.

Reply:
Is it possible for us to provide that info to them?

Reply:
> Is it possible for us to provide that info to them?

we can change the 2 texts (for error and not no NADs), what do you want to change  ?

Reply:
well if you are saying for the user who has the permissions but won't know which bridge to choose I'm asking can we tell them which network bridge to use?

Reply:
> well if you are saying for the user who has the permissions but won't know which bridge to choose I'm asking can we tell them which network bridge to use?

Sure, they should ask the cluster administrator, AFAIU this information is not available o/w

Reply:
just to clarify are you talking about network type?
<img width="677" alt="Screen Shot 2020-09-22 at 3 43 54 PM" src="https://user-images.githubusercontent.com/287880/93923927-6d07dc80-fcea-11ea-8337-9a24aa4f4b5d.png">

Reply:
@matthewcarleton no, I'm talking about bridge name (I can't see it in your screenshot)

To create a NAD you need to know about the BM host network configuration
https://docs.openshift.com/container-platform/4.3/networking/multiple_networks/configuring-host-device.html
In all the docs it's allways "As a cluster administrator, you can ..."

Reply:
ah gotcha - it sounds like we should be looking at improving that flow as well then. We should provide help to understand more about what Bridge name is if it's required. Out of scope here but relevant.

Reply:
+1 , I understand Yifat is working on QS for how to create a network

Reply:
true - hopefully we can look at just generally improving that modal too and not just rely on the quick start.

Reply:
We should state you may need to contact your cluster admin for the bridge name

Reply:
@matthewcarleton  We don't know what information is missing, it may be the bridge name or IP's in the network, host device names ... see: https://docs.openshift.com/container-platform/4.3/networking/multiple_networks/understanding-multiple-networks.html

The help texts say "... Contact your system admin for more information" , IMHO it informs the user, that if they don't know how to add a NAD without help, they probably need to contact their cluster administrator for help.

Reply:
good point. I like "Contact your system administrator for additional support" 
Sounds more helpful :)